includes/abstracts/abstract.llms.database.query.php

```diff
@@ -448,7 +542,17 @@ protected function sql_orderby() {
 			return $sql;
 		}
 
-		return apply_filters( $this->get_filter( 'orderby' ), $sql, $this );
+		/**
+		 * Filters the query ORDER BY clause
+		 *
+		 * The dynamic part of the filter `$this->id` identifies the extending query.
+		 *
+		 * @since unknown
```

Comment:
I'm confident enough with the introduction of this function with the initial method and not the 3.34.0 update.

```suggestion
		 * @since 3.8.0
```